testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/environment/EnvTwoKerberosHives.java

```diff
@@ -116,8 +114,7 @@ public void extendEnvironment(Environment.Builder builder)
     private Path createKeytabsHostDirectory()
     {
         try {
-            // Cannot use Files.createTempDirectory() because on Mac by default it uses /var/folders/ which is not visible to Docker for Mac
-            Path temporaryDirectory = Files.createDirectory(Paths.get("/tmp/keytabs-" + randomUUID().toString()));
```

Comment:
Can you run this on an intel Mac to ensure the alleged limitation is no longer present?